src/main/resources/swagger/api-docs.yaml

```diff
@@ -1475,6 +1500,27 @@ definitions:
       name:
         type: string
         description: Entity name
+  DeleteEntities:
+    properties:
+      recursive:
+        type: boolean
+        description: Whether or not to recursively delete (currently unimplemented)
+      entities:
+        type: array
+        items:
+          type: EntityID
```

Comment:
I'm having local swagger gen issues.  Do you mean that this would become:

    items:
      $ref: EntityID

Or something else?